integration-tests/tests/RegisterAddress.spec.ts

```diff
@@ -0,0 +1,60 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+
+const CREDO_PER_CTC = 1_000_000_000_000_000_000;
+const POINT_01_CTC = CREDO_PER_CTC / 100;
+
+describe('RegisterAddress', (): void => {
+    let api;
+    let alice;
+
+    beforeEach(async () => {
+        process.env.NODE_ENV = 'test';
+
+        const provider = new WsProvider('ws://127.0.0.1:9944');
+
+        api = await ApiPromise.create({ provider });
+
+        const keyring = new Keyring({ type: `sr25519` });
+        alice = keyring.addFromUri('//Alice', { name: 'Alice' });
+    });
+
+    afterEach(async () => {
+        await api.disconnect();
+    });
+
+    it('fee is min 0.01 CTC', (done): Promise<() => void> => {
```

Comment:
I've copied this syntax from an e2e PR from polkadot-js/api but it looks like the e2e test suite never got completed and is now gone. Additionally Jest v27 has changed their default test runner to jest-circus which doesn't support this syntax and the older test runner will be removed in v28. See comments after line 50.

At the moment there's no reliable way (using current Jest syntax) to execute a tx call and subscribe to the results with a callback and then assert on those results.